sql/updates/world/2018_04_19_00_world.sql

```diff
@@ -0,0 +1,7 @@
+-- fix quest 25409 Forces of Nature: Hyppogryphs
+UPDATE creature_template SET AIName='',ScriptName='npc_feralas_hippogryph_5300_5304' WHERE entry IN (5300,5304);
+UPDATE creature_template SET minlevel=36,maxlevel=37,faction=14 WHERE entry=40069;
+
```

Comment:
you change the faction to 14.. The faction comes from sniff, so not self change a value to 7 or 14.. my choose is to look at trinity or ytdb to see what value is set there.. both have all values set closely to blizz*.. both have a value of 84 so at starting our test, we use this value..

Reply:
Yes i changed the faction to make it hostile and attack player. I checked in the trinity database the, you are right, it's set to 84, previously and it was 14. Both of those factions are in a friendly state and are not attackable.

Maybe set faction in script only when start attacking. I see there is also a faction called Poacher (1086) i tried it and it attacks the player. Trinity doesn't have the quest scripted so i'm not sure what would be appropriate.

Reply:
but you have not to fight against 5300.. you select the hippo and click on the item (right upper corner) this icon click start the event..
then ?? 30%  a orc spawns and fight against you.. the hippo flight up, into the air..

Reply:
Hmmm... The script does not engage fight with 5300. The hippos (5300 & 5304) are receiving the script when the whistle item is used on them. Shortly after, the script spawns the poacher and the hippo flies off (see video https://www.youtube.com/watch?v=WWo18BHKFEk ). Sure i could have used an ItemScript instead, a different way of scripting it that would lead to the same result.

The 30% random spawn is actually 30% chance of NOT spawning, giving 70% chance. This is really more of a guess as i remember clearly that sometimes poachers did not spawn after using the item. I'm not sure how you would get the accurate chance of spawning.